Docs: Add meaningful comments to ellipses in abbreviation plugin guides. (#13332)
ckeditor/ckeditor5: docs/framework/guides/abbreviation-plugin-tutorial/abbreviation-plugin-level-3.md

export default class FormView extends View {
	constructor( locale ) {

> 		// Set the template.

Reviewer: Actually, in the final code (https://github.com/ckeditor/ckeditor5-tutorials-examples/blob/main/abbreviation-plugin/part-3/abbreviation/abbreviationview.js) above the `this.focusTracker = new FocusTracker();` is only `super( locale );`, so I would change this to "View class constructor invoke" or something like that.

		// ...

		this.focusTracker = new FocusTracker();
		this.keystrokes = new KeystrokeHandler();

> 		// Create inputs.

Reviewer: As they were multiple things here, let's go with this maybe:
```suggestion
		// Code prepared in the previous part. 
		// ...
```

		// ...
	}

    render() {

> 		// Handle the form submission.

Reviewer: ```suggestion
		// View.render() invocation and adding a submit handler. 
```

		} );

		this.setTemplate( {

> 			// Options for the form template.

Reviewer: ```suggestion
			// Setting the form template.
```

```js
// abbreviation/abbreviationui.js

// Previously imported packages.
// ...

export default class AbbreviationUI extends Plugin {
	// Previously declared methods.
	// ...

	_createFormView() {

> 		// Initialize the form view.

Reviewer: ```suggestion
		// Form view initialization. 
```

			this._hideUI();
		} );

> 		// Handle clicking outside the balloon.

Reviewer: ```suggestion
		// Handle clicking outside the balloon and on the "Cancel" button.
```

		const firstRange = selection.getFirstRange();

		if ( firstRange.isCollapsed ) {

> 			// When the selection is collapsed, the code runs.

Reviewer: We could reuse the comment from the previous step:
```suggestion
			// When the selection is collapsed, the command has a value if the caret is in an abbreviation.
```

				writer.removeSelectionAttribute( 'abbreviation' );
			} else {

> 				// When the selection is non-collapsed, the code runs.

Reviewer: Please make it a bit more descriptive,e.g. like here: https://github.com/ckeditor/ckeditor5-tutorials-examples/blob/9689e7af4903cb2a9c0f197e5a3d660dbbd4cf5f/abbreviation-plugin/part-3/abbreviation/abbreviationcommand.js#L101

				writer.removeSelectionAttribute( 'abbreviation' );
			} else {

> 				// When the selection is non-collapsed, the code runs.

Reviewer: Please make it a bit more descriptive,e.g. like here: https://github.com/ckeditor/ckeditor5-tutorials-examples/blob/9689e7af4903cb2a9c0f197e5a3d660dbbd4cf5f/abbreviation-plugin/part-3/abbreviation/abbreviationcommand.js#L101
